src/main/kotlin/com/team4099/robot2020/subsystems/Intake.kt

```diff
@@ -51,13 +54,18 @@ object Intake : Subsystem {
     }
 
     @Synchronized
-    override fun onLoop(timestamp: Double, dt: Double) {
-        synchronized(this) {
-            when (intakeState) {
-                IntakeState.IN -> intakePower = -1.0
-                IntakeState.OUT -> intakePower = 1.0
-                IntakeState.IDLE -> intakePower = 0.0
+    override fun onLoop(timestamp: Double, dT: Double) {
+
+        when (intakeState) {
+            IntakeState.IN -> {
```

Comment:
may wanna do something in `Wrist` as well to make the wrist automatically come up when ball count is greater than 5